### ISSUE TITLE
google earth on geonode

### ISSUE THREAD

**Issue by Fabolous93** [GeoNode/geonode#3940]
Hi all, i try to use google earth on geonode but i didn't do that, who can help mme? Version of geonode 2.8 on virtual machine ubuntu 16.04. Thanks you

**Comment by afabiani** [comment]
Can you please explain better the use case and what you want to achieve?

"using google earth on geonode" does not point me to any possible idea of GeoNode functionality right now.

**Comment by Fabolous93** [comment]
Yes @afabiani. I saw the Geonode's (2.8 master) manual in this part:

![googleearthplugin](https://user-images.githubusercontent.com/42996391/46676271-74f31280-cbe0-11e8-80e9-693222dca4a8.png)
 
How to insert Google Earth plugin with icon connected?
Maybe i can add into Geonode map composer with not this version, but older

**Comment by afabiani** [comment]
Hi @Fabolous93, unfortunately, Google dismissed that feature a lot of time ago.

Integrated GoogleEarth on browsers won't be available anymore. If you just need 3D visualization, though, there are new REACT based gis clients with Cesium which would allow you to achieve that.
